docs/exts/eventsub.rst

```diff
@@ -539,6 +543,12 @@ API Reference
 
 .. attributetable::: ChannelModerateData
 
+.. autoclass:: ChannelAdBreakBeginData
+    :members:
+    :inherited-members:
+
+.. attributetable::: ChannelAdBreakBeginData
```

Comment:
This is going to break the docs. `ChannelAdBreakBeginData` should be on top of it's autoclass and same with `ChannelModerateData`